main.py

```diff
@@ -272,9 +272,10 @@ def process(self, world_object: World) -> None:
             enemies = [enemy for enemy in world_object.worms_at(worm.coordinates)
                        if enemy is not worm]
             affordable_food = world_object.food_at(worm.coordinates)
-            if worm.is_dangerous(
-                    worm.max_danger_at_location(world_object.worms_at(worm.coordinates))
-            ) or (len(affordable_food) == 0 and len(enemies) == 0):
+            worm_in_danger = worm.is_dangerous(
+                worm.max_danger_at_location(world_object.worms_at(worm.coordinates)))
+
+            if worm_in_danger or (len(affordable_food) == 0 and len(enemies) == 0):
```

Comment:
Second part of condition: len(affordable_food) == 0 and len(enemies) == 0 you can pre-cache in boolean variable like **is_nothing_interesting_in_cell** or so.